modules/scheduler/scheduler/src/main/resources/webapp/js/controllers.js

```diff
@@ -209,6 +209,14 @@
             return date;
         }
 
+        $scope.getMinDate = function(jobType) {
```

Comment:
Why do we only enforce starting date for run once jobs? Is this for convenience when updating jobs?